### DISCUSSION TITLE
Unavailable repo since 8 hours

### DISCUSSION THREAD

**Discussion by HusseinSalami** [community/community#17516] [Other Feature Feedback, Questions, & Ideas]
I'm getting "this repository is temporarily unavailable. This repository is moving to a newer, better home. It’ll be back in just a few minutes." Any recommendation?

**Comment by queenofcorgis** [comment]
Hi there - are you still experiencing this?

**Reply by HusseinSalami** [reply]
No, it works now. The support team unlocked it.
Thank you for the follow up!

**Reply by mahanmoulaei** [reply]
> Hi there - are you still experiencing this?

@queenofcorgis Hello, I'm facing the same problem with one of my organization's repositories for over 48 hours now. Can you guide what to do, please?

**Comment by HusseinSalami** [comment]
Hey Mahan,

I had to open a case to github support team to fix the issue.

I wish it can help you.

Hussein



Sent from my Galaxy



-------- Original message --------
From: Mahan Moulaei ***@***.***>
Date: 21/02/2023 01:29 (GMT+02:00)
To: community/community ***@***.***>
Cc: HusseinSalami ***@***.***>, Author ***@***.***>
Subject: Re: [community/community] Unavailable repo since 8 hours (Discussion #17516)


Hi there - are you still experiencing this?

@queenofcorgis<https://github.com/queenofcorgis> Hello, I'm facing the same problem with one of my organization's repositories for over 48 hours now. Can you guide what do, please?

**Reply by mahanmoulaei** [reply]
Hi Hussein,
Thanks for the reply. 👍

**Reply by HusseinSalami** [reply]
@altafNEU , it was quite quick. Within one week i had it fixed.

**Comment by altafNEU** [comment]


**Reply by HusseinSalami** [reply]
@altafNEU , it was quite quick. Within one week i had it fixed.
